modules/account-lib/src/coin/baseCoin/iface.ts

```diff
@@ -130,3 +130,29 @@ export type FeeOptions = {
   formula?: 'fixed' | 'feeRate' | 'perKB' | 'custom';
   type?: 'base' | 'max' | 'tip';
 };
+
+/**
+ * The validity window signals when a transaction can be included in the network.
+ *  ValidityWindow information is stored as part of the transaction data in the indexer and wallet platform, unless it is absolutely necessary.
```

Comment:
firstValid: the absolute or relative block/time after which the transaction is valid
lastValid: the absolute or relative block/time before which the transaction is valid
minDuration: minimum duration a transaction for this coin is valid
maxDuration: maximum duration a transaction for this coin is valid

Reply:
minDuration and maxDuration are defined per blockchain, whereas firstValid and lastValid are defined per transaction